scripts/build-pages.js

```diff
@@ -0,0 +1,47 @@
+const path = require("path");
+const execa = require("execa");
+const glob = require("glob").sync;
+const fs = require('fs').promises;
+const fsSync = require('fs');
+
+const enginePath = path.resolve(__dirname, "../packages/ReactDagEditor");
+const demoSrc = path.join(enginePath, '.static-demo');
+const demoDist = path.resolve(__dirname, '../docs');
+const gitIgnorePath = path.resolve(__dirname, '../.gitignore');
+
+async function buildDemo() {
```

Comment:
I see there is a npm package: https://www.npmjs.com/package/gh-pages to do the copy-paste and branch management